docs/developer/decision-records/2024-03-06-bigquery/README.md

```diff
@@ -0,0 +1,30 @@
+# BigQuery support for EDC
+
+## Decision
+
+Add the BigQuery extensions to enable the transfer of data to and from BigQuery tables.
+
+### Extensions
```

Comment:
nit: most of our other decision records have the following structure:
- Decision: states _what_ has been decided. you already have that here.
- Rationale: _why_ and _how_ that decision was reached, i.e. paraphrasing "because we want to support a wider ranger of GCP services, namele BigQuery"
- Approach: how the decision is put into action. Basically, just put your "Extensions", "Data Source", "Data Sink", "Notes", and "Future Development" sections under a the "Approach" heading.

Reply:
Done